lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb

```diff
@@ -302,12 +302,12 @@ def in_range(range)
           self
         end
 
-        def allow_blank(allow_blank = true)
+        def allow_blank
           @options[:allow_blank] = allow_blank
```

Comment:
Does this need to be `true`? It seems like this would create an infinite recursive loop.